modules/data.remote/R/MODIS_LC_prep.R

```diff
@@ -0,0 +1,178 @@
+#' Prepare MODIS land cover data for the SDA workflow.
+#'
+#' @param site_info Bety list of site info including site_id, lon, and lat.
+#' @param time_points A vector contains each time point within the start and end date.
+#' @param outdir Where the final CSV file will be stored.
+#' @param export_csv Decide if we want to export the CSV file.
+#' @param qc.filter decide if we want to filter data by the qc band.
```

Comment:
1. recommend qc_filter as the variable name
2. similar to earlier point, rather than having this be a boolean, better would be to pass in the requested QC flag (e.g. set this to qc_filter = c("000", "001") by default) and then say set to NULL or FALSE or something like that to turn off QC filtering.

Reply:
fixed.